lib/generators/react_on_rails/templates/base/base/package.json

```diff
@@ -29,5 +29,6 @@
     "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
   },
   "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
-  "dependencies": {}
+  "dependencies": {},
+  "cacheDirectories": ["node_modules", "client/node_modules"]
```

Comment:
@modosc  there should be no node_modules directory, right? Just client/node_modules.